exporter/fileexporter/README.md

```diff
@@ -47,6 +47,7 @@ The following settings are optional:
   - localtime : [default: false (use UTC)] whether or not the timestamps in backup files is formatted according to the host's local time.
 
 - `format`[default: json]: define the data format of encoded telemetry data. The setting can be overridden with `proto`.
+- `mode`[default: truncate] defines whether to truncate or append to the file. The setting can be overridden with `append`.
```

Comment:
The configuration property name `mode` sounds a bit vague to me.

I like the simplicity of `append: true|false`. Are there any other options we might consider in future other than `append` and `truncate`?

Reply:
We agreed on going forward with the boolean implementation.